README.md

```diff
@@ -58,7 +58,7 @@ scrape_configs:
       module: [http_2xx]  # Look for a HTTP 200 response.
     target_groups:
       - targets:
-        - mywebsite.com   # Target to probe
+        - www.prometheus.io   # Target to probe
```

Comment:
prometheus.io is the canonical version. Next prometheus release we can put the http:// in front too.